python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -609,6 +623,11 @@ cdef class ParquetWriter:
         self.row_group_size_rows = row_group_size_rows
         self.max_page_size_bytes = max_page_size_bytes
         self.max_page_size_rows = max_page_size_rows
+        self.max_dictionary_size = (
```

Comment:
shouldn't this assign to `dict_policy`?

Reply:
Yes, my bad. I am amazed that it compiled fine.

Reply:
I was surprised as well. That's what prompted me to suggest tests :D

Reply:
😆 makes sense.